tests/test_cron_workflow.py

```diff
@@ -134,5 +136,16 @@ def test_cwf_overwrites_head_and_tail(cw, no_op):
 def test_cwf_valid_field_set(cws):
     cw = CronWorkflow('cw', "* * * * *", service=cws, parallelism=33)
     assert cw.schedule == "* * * * *"
+    assert cw.timezone is None
     assert cw.service == cws
     assert cw.parallelism == 33
+
+
+def test_cwf_valid_timezone_set(cws):
+    cw = CronWorkflow('cw', "* * * * *", timezone="UTC", service=cws)
+    assert cw.timezone == "UTC"
+
+
+def test_cwf_invalid_timezone_set(cws):
+    with pytest.raises(ValueError):
+        cw = CronWorkflow('cw', "* * * * *", timezone="foobar", service=cws)
```

Comment:
Perhaps include a check for the error message to ensure we get a failing test in case it changes? 
```suggestion
def test_cwf_invalid_timezone_set(cws):
    with pytest.raises(ValueError) as e:
        cw = CronWorkflow('cw', "* * * * *", timezone="foobar", service=cws)
    assert str(e) == 'foobar is not a valid timezone'
```